### PR TITLE
CCHS-184: Create new SLD for polygon data

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

**Comment by zmoore-usgs** [comment]
Currently working on addressing feedback from: https://github.com/USGS-CIDA/coastal-hazards/pull/1591

File: coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/SLD/evic.jsp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/SLD/evic.jsp]: Is this just a copy of bins_polygon.jsp?  There isn't a problem if EVIC uses polygon, it just needs to link to it in the SLDConfig.

Comment (zmoore-usgs) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/SLD/evic.jsp]: It has some tweaks. bins_polygon uses the thresholds directly to calculate things and seems to have a lot more going on. This uses the defined upper and lower bounds that are calculated in the SLDConfig

Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/SLD/evic.jsp]: So is bins_polygon used anywhere?

Comment (zmoore-usgs) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/SLD/evic.jsp]: Doesn't look like it, so I could just edit that I suppose.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/Evic.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import static gov.usgs.cida.coastalhazards.Attributes.*;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ * @author Kathryn Schoephoester <kmschoep@usgs.gov>
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/Evic.java]: Copypasting?

Comment (mhines-usgs) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/Evic.java]: no, Kathy actually did this...

Comment (isuftin) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/Evic.java]: Right but why is Zack checking it in?

Comment (isuftin) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/Evic.java]: Ah I see now. @kmschoep-usgs needs a recognizable avatar 🥇 D

Comment (mhines-usgs) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/Evic.java]: get back to work ;) :shoo:

**Pull request merged**